src/main/kotlin/com/openlattice/indexing/BackgroundIndexingService.kt

```diff
@@ -238,7 +245,7 @@ class BackgroundIndexingService(
 
     }
 
-    private fun tryLockEntitySet(entitySet: EntitySet): Boolean {
+    internal fun tryLockEntitySet(entitySet: EntitySet): Boolean {
```

Comment:
Shouldn't we use this func to lock entitySets in IndexingService?

Reply:
nah, explicitly decided to ignore locking of entity sets for reindexer.